plugins/channel_header_plug/channel_header_plug.jsx

```diff
@@ -117,9 +117,13 @@ export default class ChannelHeaderPlug extends React.PureComponent {
     }
 
     createButton = (plug) => {
+        let activeClass = '';
```

Comment:
Can we add test to cover the `active` and `show` in the `plugins/channel_header_plug/channel_header_plug.test.jsx` file?